net/sshuttle/files/sshuttle.init

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+# shellcheck disable=SC2034
+START=99
+STOP=01
+USE_PROCD=1
+
+_log() {
+  logger -p daemon.info -t sshuttle "$@"
```

Comment:
Suggest you switch to tabs for indentation; disk space is a scarce resource for OpenWrt devices.

Reply:
sure, done